doc/source/how-to/grpc-api-packages.rst

```diff
@@ -245,28 +245,9 @@ replaced by:
 Consuming the API package within Python
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once the API package has been published to PyPI, a reference can be included within
-the client library build dependencies.
-
-.. tab-set::
-
-    .. tab-item:: flit
-
-        .. code-block:: toml
-
-            [project]
-            dependencies = [
-              ansys-api-service==X.Y.Z
-            ]
-
-    .. tab-item:: poetry
-
-        .. code-block:: toml
-
-            [tool.poetry.dependencies]
-            python = ">=3.7,<4.0"
-            ansys-api-service = "==X.Y.Z"
-
+Once the API package has been published to PyPI, a reference can be included
+within the client library build dependencies. To know how to specify project
+required dependencies, refer to the :ref:`Required Dependencies` section.
```

Comment:
```suggestion
dependencies, see :ref:`Required Dependencies`.
```

Reply:
The title should be sentence case, so "Required dependenices". However, I didn't want to break the link.